### PR TITLE
 Replay test with 12_1_0_pre3

### PR DESCRIPTION
**Pull request by qliphy**
See #4605 

Runs: 341169,341754,342154,344518

Release: CMSSW_12_1_0_pre3

GT: 
- Express: 113X_dataRun3_Express_v4
- Prompt: 113X_dataRun3_Prompt_v3

### CODE REVIEW

**Comment by qliphy** [comment]
@germanfgv   I noticed "Syntax Check" and "Unit Tests" above show failure message but not obvious details.  Can I just ignore them?

**Comment by germanfgv** [comment]
@qliphy we are still updating our test to run with python3. They still don't work perfectly with our new agent. So yes, they can be ignored. I'll manually the replay and let you know about the results.

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_11_3_2"
+    'default': "CMSSW_12_1_0_pre3"
```
Comment (francescobrivio) [inline etc/ReplayOfflineConfiguration.py:103]: @qliphy shouldn't you also add the cmssw override for repacking (line 145) and express (line 159)?

Comment (qliphy) [reply etc/ReplayOfflineConfiguration.py:103]: @francescobrivio   Thanks. Now added.

**Comment by germanfgv** [comment]
@qliphy @francescobrivio after this changes were made, we restarted the replay with the proper config. Around 50% of the jobs have finished with no issues so far.

**Comment by germanfgv** [comment]
As reported in [HyperNewas](https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/2266.html?inline=-1), the replay was successfull with the current configuration. I'll merge this as the current default replay config for Tier0.

**Pull request merged**